flask_resty/testing.py

```diff
@@ -1,12 +1,36 @@
+from datetime import datetime, timedelta
+import re
+
+from dateutil import parser
```

Comment:
Well, what do you want to do? `any_dict.containing(1).not_containing(2)` does feel a bit different from what we have.

Let's stick with this for now, minus the date stuff? And then figure out what to do further later.